packages/web/src/scss/components/Tabs/_Tabs.scss

```diff
@@ -1,14 +1,30 @@
 @use '../../tools/reset';
 @use '../../tools/typography';
+@use '../../tools/breakpoint';
 @use 'theme';
 
 .Tabs {
     display: flex;
     flex: none;
-    gap: theme.$tab-gap;
+    gap: var(--tabs-gap);
     align-items: center;
     justify-content: center;
     list-style: none;
+
+    $property-name: --tabs-spacing;
```

Comment:
I think we need to reset these CSS variable values for children elements. If it happened and we had nested tabs, it would interfere.

Reply:
Nested tabs? 🤯

Reply:
Anyway, I think this is covered below (lines 33–40).